en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-configuration.md

```diff
@@ -0,0 +1,111 @@
+# Amazon Lambda Configuration
+
+The AmazonLambda Connector allows you to access the REST API of [Amazon Web Service Lambda (AWS Lambda)](https://docs.aws.amazon.com/lambda/latest/dg/welcome.html), which lets you run code without provisioning or managing servers. With Lambda, you can run code for virtually any type of application or backend service - all with zero administration. Just upload your code in one of the languages that AWS Lambda supports (currently Node.js, Java, C#, Go, and Python) and Lambda takes care of everything required to run and scale your code with high availability. You can set up your code to automatically trigger from other AWS services or call it directly from any web or mobile app. You pay only for the compute time you consume - there is no charge when your code is not running.
+
+To use the AmazonLambda service, you must have an AWS account. If you don't already have an account, you are prompted to create one when you sign up. You're not charged for any AWS services that you sign up for unless you use them.
+
+## Signing Up for AWS
+
+* **To sign up for AWS:**
+
+    1. Navigate to [Amazon AWS website](https://aws.amazon.com/), and then select **Create an AWS Account**.
+
+        > **Note**: If you previously signed in to the AWS Management Console using AWS account root user credentials, select **Sign in to a different account**. If you previously signed in to the console using IAM credentials, choose Sign-in using root account credentials. Then select **Create a new AWS account**.
+
+    2. Follow the online instructions.
+
+Part of the sign-up procedure involves receiving a phone call and entering a verification code using the phone keypad. AWS will notify you by email when your account is active and available for you to use.
+
+## Obtaining user credentials
+
+You can access the Amazon Lambda service using the root user credentials but these credentials allow full access to all resources in the account as you cannot restrict permission for root user credentials. If you want to restrict certain resources and allow controlled access to AWS services then you can create IAM (Identity and Access Management) users in your AWS account for that scenario.
+* **Follow the steps below to get an AWS Access Key for your AWS root account:**
```

Comment:
Let's remove the bullet here and make this like a subsection. Same applies for the bullet below.

Reply:
Bullets points are updated as two subsections.